material-ui/material-ui.d.ts

```diff
@@ -1140,7 +1146,7 @@ declare namespace __MaterialUI {
         export class TableHeaderColumn extends React.Component<TableHeaderColumnProps, {}> {
         }
 
-        interface TableRowProps extends React.Props<TableRow> {
+        interface TableRowProps extends HasStyleAndClassNameProps<TableRow> {
```

Comment:
Based on the `render()` method in the material-ui source code, since `tr` is passed all `other`  properties, use `React.HTMLAttributesBase<TableRow>` instead of `HasStyleAndClassNameProps`.